README.md

```diff
@@ -8,6 +8,8 @@ Gatsby Starter using the [Emulsify Gatsby Theme](https://github.com/fourkitchens
 
 ## 🚀 Install
 
+`composer install`
+
 `yarn` or `npm install`
```

Comment:
Should we consider rolling this into the `composer install` command? I know this detours off of the node install process from the base emulsify-gatsby package but this would simplify the process of running the install.

Reply:
I would really rather this stay a node install if possible - there are no PHP dependencies anymore for this project outside of when it is installed in Drupal. For Drupal-specific developers, we would have Drupal-specific instructions, which would obviously be a composer install. Is that OK?